src/get-or-create-render-context.js

```diff
@@ -7,53 +7,53 @@
 const vm = require("vm");
 // TODO(csilvers): try to get rid of the dependency on jsdom
 const jsdom = require("jsdom");
-const raf = require("raf");
 const cache = require("./cache.js");
 const profile = require("./profile.js");
 
-const runInContext = function(context, fn) {
-    return vm.runInContext("(" + fn.toString() + ")()", context);
-};
+class CustomResourceLoader extends jsdom.ResourceLoader {
```

Comment:
This is a very simple custom loader that adds a line to the server console so we can see what files get dynamically requested.